modules/govuk_jenkins/templates/jobs/departmental_projects_content_change_to_draft.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- scm:
+    name: departmental_projects_content_change_to_draft
```

Comment:
I believe this name needs to be different from the job name - check the other templates as examples.